language/de/acp/ban.php

```diff
@@ -77,11 +77,11 @@
 	'PERMANENT'		=> 'Dauerhaft',
 
 	'UNTIL'						=> 'Bis zum',
-	'USER_BAN'					=> 'Einen oder mehrere Benutzernamen sperren',
+	'USER_BAN'					=> 'Einen oder mehrere Benutzer mittels Benutzernamen sperren',
```

Comment:
Die Klammer war hier mehr als Hinweis gedacht, dass man den Teil in Klammern vermutlich problemlos weglassen kann.